src/ec/packages/ec-component-timeline/timeline.test.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-param-reassign */
+
+import path from 'path';
+import { merge, renderTwigFileAsNode } from '@ecl-twig/test-utils';
+
+import demoData from './demo/data';
+
+describe('EC - Timeline', () => {
+  const template = path.resolve(__dirname, './timeline.html.twig');
+  const render = params => renderTwigFileAsNode(template, params);
+
+  test('renders correctly without hidden items', () => {
+    expect.assertions(1);
+
+    return expect(render(demoData)).resolves.toMatchSnapshot();
+  });
+
+  test('renders correctly with hidden items', () => {
+    expect.assertions(1);
+
+    const optionsWithExtraClasses = merge(demoData, {
+      hide: {
+        from: 10,
+        to: -2,
+      },
+    });
+
+    return expect(render(optionsWithExtraClasses)).resolves.toMatchSnapshot();
+  });
+
+  test('renders correctly with hidden items but no items after button', () => {
+    expect.assertions(1);
+
+    const optionsWithExtraClasses = merge(demoData, {
+      hide: {
+        from: 10,
+        to: 0,
```

Comment:
would be nice to have a test scenario is a positive `hide.to` or with a negative `hide.from` :)